tools/dynamo/verify_dynamo.py

```diff
@@ -81,7 +81,7 @@ def get_rocm_version():
 def check_cuda():
     import torch
 
-    if not torch.cuda.is_available():
+    if not torch.cuda.is_available() or torch.version.cuda is None:
```

Comment:
Did we plan to float this as a standalone PR?

Reply:
That PR is up now
https://github.com/pytorch/pytorch/pull/97013